code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -175,28 +171,24 @@
 		holder_type = species.holder_type
 
 	icon_state = lowertext(species.name)
-
 	species.handle_post_spawn(src)
-
 	maxHealth = species.total_health
 
-
 	spawn(0)
 		regenerate_icons()
 		make_blood()
 
 	// Rebuild the HUD. If they aren't logged in then login() should reinstantiate it for them.
-	if(client && client.screen)
+	if(client?.screen)
 		client.screen.len = null
 		if(hud_used)
 			qdel(hud_used)
 		hud_used = new /datum/hud(src)
 
-
 	if(species)
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
```

Comment:
Could replace 188-191 with

return !!species

Not sure if you'd consider it more readable, I do that lots though. *shrug